app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -305,6 +304,16 @@ def handle_expired_status_update(enrollment, response, response_message)
     end
   end
 
+  # MW: What else do we want to log here?
+  def handle_fraud_review_pending(enrollment)
+    enrollment.profile&.deactivate_for_fraud_review
+
+    analytics(user: enrollment.user).
+      idv_in_person_usps_proofing_results_job_user_sent_to_fraud_review(
+        **enrollment_analytics_attributes(enrollment, complete: true)
+      )
```

Comment:
i believe we get the uuid for free from here: https://github.com/18F/identity-idp/blob/33594ec1db8f2da83364ac5d44b8645d8413b4a2/app/services/analytics.rb#L25

Reply:
This has been tricky to test, because the test setup stubs out the call to `analytics` where the user parameter is passed in, and it seems like the `have_received_event` (or similar?) matcher is not looking at what's passed into the constructor, so the `user` parameter never shows up.

Reply:
It is being logged by the code Shannon points out. I [started to fix the FakeAnalytics code](https://github.com/18F/identity-idp/pull/9928#issuecomment-1912448226), but am going to tackle that separately because it breaks a lot.